src/commands/deploy/metadata.ts

```diff
@@ -99,6 +100,7 @@ export default class DeployMetadata extends SfCommand<DeployResultJson> {
       char: 'o',
       description: messages.getMessage('flags.target-org.description'),
       summary: messages.getMessage('flags.target-org.summary'),
+      required: true,
```

Comment:
Why add required true here? Should be unnecessary?

Reply:
yeah, that's how I wish it worked.  Without it, and with strict null checks, oclif flag typing isn't sure whether it could be undefined.  We can maybe remove it once we get oclif fixed (hopefully Mike's oclie/core v2)